WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -32,6 +32,11 @@
           get :index, event_ids: ["777", "333fm", "333oh"]
           expect(assigns(:competitions)).to eq [competition4]
         end
+
+        it "works when event_ids are passed as hash instead of an array (facebook redirection)" do
```

Comment:
Can you add a link back to https://github.com/cubing/worldcubeassociation.org/issues/472?